src/trinary_sse42.h

```diff
@@ -72,4 +73,161 @@ static inline bool validateTrytes_sse42(Trobject_t *trytes)
     return true;
 }
 
+static inline Trobject_t *trytes_from_trits_sse42(Trobject_t *trits)
+{
+    Trobject_t *trytes = NULL;
+    int8_t *src = (int8_t *) malloc(trits->len / 3);
+
+    const int block_8bit = BLOCK_8BIT(__m128i);
+    int8_t setMSB = 0x80;
+    __m128i tryteAlphabet[2] = {
+        _mm_setr_epi8('9', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J',
```

Comment:
Why did you enumerate the alphabet between 'A' to 'Z'?

Reply:
For the later use of transforming the trit to tryte.
It is similar to the `TryteAlphabet` defined in `src/constants.c`.

Reply:
Is it possible to introduce macros for such enumeration?

Reply:
Can I have more clues that the code modification should probably look like?

Reply:
https://github.com/jserv/sse2neon/blob/master/sse2neon.h # Check macro `_MM_SHUFFLE`

Reply:
I only substituted them with the variable `TryteAlphabet`.